runtime/doc/filetype.txt

```diff
@@ -513,40 +513,40 @@ Local mappings:
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
 View manpages in Nvim. Supports highlighting, autocompletion, locales, and
-navigation. See also |find-manpage|.
+navigation. Also see |find-manpage|.
```

Comment:
"See also" is pretty typical wording.  Either works, but I would wager that "see also" is more common.